paddle/cuda/src/hl_cuda_sparse.cu

```diff
@@ -562,6 +562,22 @@ void hl_memcpy_sparse_matrix(hl_sparse_matrix_s dst,
   }
 }
 
+/**
+ * Calculate beta * C, if beta is zero, C does not have to be a valid input.
+ */
+static void _beat_mul_c(real *c, int dimM, int dimN, real beta) {
+  if (beta == 0.0) {
+    hl_gpu_apply_unary_op(unary::Zero<real>(), c, dimM, dimN, dimN);
```

Comment:
I see the "=" is used in the kernel if beta == 0, namely
```
+__device__ __forceinline__
 +void _calculate_c(real &c, real sum) {
 +  c = sum;
 +}
```
so could this zero memory operation be removed? Otherwise, it will zero every time, although it may not be consuming.